diff --git a/bot.py b/bot.py
index 0c748e5..2ff571b 100644
--- a/bot.py
+++ b/bot.py
@@ -789,6 +789,12 @@ async def ocr_screenshot(message):
     msg = message.content.lower()
     msg = msg.replace("ocr ", "").replace("ocr", "")
     msg = msg.split(" ")
+    
+    if " post" in message.content:
+        message.content = message.content.replace(" post", "")
+        post = True
+    else:
+        post = False
 
     correction = False
     if "+" in message.content:
@@ -815,7 +821,7 @@ async def ocr_screenshot(message):
         with open(fname, "wb") as f:
             f.write(img.content)
         try:
-            result = AC_Score_OCR.OCR(fname, game, players)
+            result = AC_Score_OCR.OCR(fname, game, players, post)
         except:
             result = "Sorry, something went wrong with your screenshot. We recommend using mpv to take screenshots."
         if correction:

